packages/backend/src/app/workers/sandbox/file-sandbox.ts

```diff
@@ -35,7 +35,7 @@ export class FileSandbox extends AbstractSandbox {
         const command = [
             `cd ${this.getSandboxFolderPath()}`,
             '&&',
-            `cross-env-shell AP_PIECES_SOURCE=${pieceSources} NODE_OPTIONS=--enable-source-maps`,
+            `cross-env-shell AP_PIECES_SOURCE=${pieceSources} NODE_OPTIONS=--enable-source-maps ${process.platform === 'win32' ? '&&' : ''}`,
```

Comment:
Windows works in mysterious way, I am curious how && solved it.

Reply:
Do you have to apply it for package.json aliases too?

Reply:
No npm start works fine, but this is needed here for some reason :D